sdk/cosmos/tests/setup.rs

```diff
@@ -24,8 +30,15 @@ pub fn initialize(
         == Ok(azure_core::TESTING_MODE_RECORD))
     .then(get_account)
     .unwrap_or_else(String::new);
+    let authorization_token = (std::env::var(azure_core::TESTING_MODE_KEY).as_deref()
+        == Ok(azure_core::TESTING_MODE_RECORD))
+    .then(|| get_authorization_token().ok())
```

Comment:
Might be better for us to just unwrap the error. If the testing mode is set to record but we can't parse the auth token, that's a bug.